enterprise/frontend/src/embedding-sdk/components/private/InteractiveQuestion/components/Breakout/Breakout.tsx

```diff
@@ -25,11 +24,8 @@ const AddBreakoutPopover = () => {
   );
 };
 
-export const BreakoutInner = ({
-  question,
-  updateQuestion,
-}: QuestionStateParams) => {
-  const breakoutItems = useBreakoutData({ question, updateQuestion });
+export const BreakoutInner = () => {
+  const breakoutItems = useBreakoutData();
```

Comment:
Make the API consistent with other 2 hooks.
- `useFilterData` and
- `useSummarizeData`

This is also to make MetabotQuestion not have to pass `question` and `updateQuestion` to the hook. So the code is more consistent.